src/library/browse/browsefeature.cpp

```diff
@@ -344,6 +346,11 @@ void BrowseFeature::onLazyChildExpandation(const QModelIndex& index) {
                 drive.filePath()); // Displays C:/
             folders << driveLetter;
         }
+#elif defined(__APPLE__)
+        qWarning() << "Trying to process DEVICE_NODE on macOS; we shouldn't reach this point!";
```

Comment:
This can become a: 
DEBUG_ASSERT(!"Trying to process DEVICE_NODE on macOS");

Reply:
Done.